### PR TITLE
Fix warning when the emission is deleted

### PR DESCRIPTION
**Pull request by Jeconias**

### CODE REVIEW

File: app/screens/EmissionItem/EmissionItemScreen.tsx [hunk 1]
```diff
@@ -35,60 +35,75 @@ const EmissionItemScreen = ({ language = "" }: LocalizationContextProps) => {
...
 
   const date = moment(creationDate, "YYYY-MM-DDTHH:mm:ss.sssZ");
   const day = date.locale(language).format("dddd");
   const monthAndYear = date.locale(language).format("Do MMMM YYYY");
-  const co2Emission = calculation.getC02ValueFromEmission(emission);
-  const deleteEmission = () =>
+  const co2Emission = calculation.getC02ValueFromEmission(emission || {});
+  const deleteEmission = () => {
     dispatch(emissions.actions.deleteEmissionById(emission.id));
+    navigator.goBack();
```
Comment (PierreBresson) [inline app/screens/EmissionItem/EmissionItemScreen.tsx:50]: Delete this line otherwise we have a double goBack because of the useEffect

**Review by PierreBresson — CHANGES_REQUESTED** [review]
just a little change and we are good to go

File: app/screens/EmissionItem/EmissionItemScreen.tsx (hunk 1 above)
Comment (Jeconias) [reply app/screens/EmissionItem/EmissionItemScreen.tsx]: Done!

**Comment by PierreBresson** [comment]
@Jeconias muito obrigado ✌️

**Pull request merged**